### PR TITLE
Update feature-gates.md

### PR DESCRIPTION
**Pull request by lf1029698952**
Update PodPriority feature to beta since v1.11

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
here ref:
	// owner: @bsalamat
	// alpha: v1.8
	// beta: v1.11
	// GA: v1.14

**Comment by zhangxiaoyu-zidif** [comment]
if you want to merge this PR to master branch, it should be GA now.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -104,7 +104,8 @@ different Kubernetes components.
 | `NodeLease` | `false` | Alpha | 1.12 | |
 | `PersistentLocalVolumes` | `false` | Alpha | 1.7 | 1.9 |
 | `PersistentLocalVolumes` | `true` | Beta | 1.10 | |
-| `PodPriority` | `false` | Alpha | 1.8 | |
+| `PodPriority` | `false` | Alpha | 1.8 | 1.10 |
+| `PodPriority` | `true` | Beta | 1.11 | |
```
Comment (bsalamat) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: PodPriority graduated to beta in 1.11 and was in beta till 1.13. It graduated to GA in 1.14.

**Pull request merged**